src/test/scala/is/hail/expr/ir/LocusFunctionsSuite.scala

```diff
@@ -52,4 +54,9 @@ class LocusFunctionsSuite extends TestNGSuite {
   @Test def inYPar() {
     assertEvalsTo(invoke("inYPar", locusIR), locus.inYPar(grch38))
   }
+
+  @Test def minRep() {
+    val alleles = MakeArray(Seq(Str("A"), Str("T")), TArray(TString()))
+    assertEvalsTo(invoke("min_rep", locusIR, alleles), Row(Locus("chr22", 1), IndexedSeq("A", "T")))
```

Comment:
Can you use an example instead that doesn't return the exact input? See `test_expr` for examples. Also, null values at either the locus and alleles.